platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -114,6 +133,15 @@ struct ipsec_sa_s {
 		};
 	} hot;
 
+	/* Anti-replay window management. */
+	struct {
+		/* Number of buckets in AR window */
+		uint16_t nb_bucket;
+
+		/* AR window size  */
+		uint32_t win_size;
+	} ar;
```

Comment:
Since these are needed only for inbound SAs, they could be moved to ipsec_sa_t::in (i.e. in the same struct where lookup_ver is).

I think num_buckets would be a better name than nb_bucket. The comment should not say "in window" since not all the buckets are necessarily inside the window (when the window boundary coincides with a bucket boundary, there is
at least one bucket that is not "in the window"). But "for AR window" would sound ok to me.

Reply:
Ack, will make these changes.